src/templates.ts

```diff
@@ -110,16 +110,36 @@ export function getTemplates(options: ModuleOptions, uiConfig: Record<string, an
   writeThemeTemplate(theme)
 
   async function generateSources() {
-    let sources = ''
+    if (!nuxt) {
+      return '@source "./ui";'
+    }
+
+    const sources: string[] = []
+    const layers = getLayerDirectories(nuxt).map(layer => layer.app)
+    const extensions = nuxt.options.extensions.map(ext => ext.replace(/^\./, '')).join(',')
+
+    // Add layer sources
+    for (const layer of layers) {
+      sources.push(`@source "${layer}**/*.{${extensions}}";`)
+    }
```

Comment:
```suggestion
    const extensions = nuxt.options.extensions.join(',')

    // Add layer sources
    for (const layer of layers) {
      sources.push(`@source "${layer}**/*{${extensions}}";`)
    }
```

This may not be that relevant, but this is the approach of Daniels Knip PR:https://github.com/webpro-nl/knip/pull/1265/files#diff-94cc8dab409c9b7cd7fc21a94d45b1fd23f3fd3c5eb2a152dea271fa1bbeff59R90

Reply:
Mmmh I'm wondering if this will work in Tailwind CSS though. I'm not even sure we need the extensions actually: https://tailwindcss.com/docs/detecting-classes-in-source-files#explicitly-registering-sources